idaes/models/unit_models/separator.py

```diff
@@ -209,6 +210,22 @@ class SeparatorData(UnitModelBlockData):
 **MaterialBalanceType.total** - use total material balance.}""",
         ),
     )
+    CONFIG.declare(
+        "momentum_balance_type",
+        ConfigValue(
+            default=MomentumBalanceType.pressureTotal,
+            domain=In(MomentumBalanceType),
+            description="Momentum balance construction flag",
+            doc="""Indicates what type of momentum balance should be constructed,
+    **default** - MomentumBalanceType.pressureTotal.
+    **Valid values:** {
+    **MomentumBalanceType.none** - exclude momentum balances,
+    **MomentumBalanceType.pressureTotal** - single pressure balance for material (pressure in all outlets is equivalent),
```

Comment:
I think we should just make these say "pressure in all outlets is equal". It might be even more correct to make anything but `none` and `pressureTotal` unsupported for now.

Reply:
I've updated. Since I click request your review again, the other reviewers disappeared, so please help me to add them again, thank you!